apps/emqx_management/src/emqx_mgmt_api_cluster.erl

```diff
@@ -134,5 +207,15 @@ force_leave(delete, #{bindings := #{node := Node0}}) ->
 join(Node) ->
     ekka:join(Node).
 
+-spec connected_replicants() -> [node()].
+connected_replicants() ->
+    maps:fold(
+        fun(K, V, Acc) ->
+            [#{node => K, streams => length(V)} | Acc]
+        end,
+        [],
+        maps:groups_from_list(fun({_, N, _}) -> N end, mria_status:agents())
```

Comment:
Might be a bit more future-proof to respond with whole `mria_status:agents()` here? 🤔 And do the aggregation in the API handler.

Reply:
Good idea, thanks.
Fixed.